<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Formula</class>
 <widget class="QWidget" name="Formula">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_39">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>370</y>
     <width>100</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>80</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#F1F5FE;
border-style:solid; /*边框风格*/
border-width:1px;/*边框宽度*/
border-color:#B4B6BC; /*边框颜色*/
border-radius:10px; /*边框倒角*/</string>
   </property>
   <property name="text">
    <string>垛盘 3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1024</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
	border:  1px solid black;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>配方参数</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_37">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>100</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>80</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#F1F5FE;
border-style:solid; /*边框风格*/
border-width:1px;/*边框宽度*/
border-color:#B4B6BC; /*边框颜色*/
border-radius:10px; /*边框倒角*/</string>
   </property>
   <property name="text">
    <string>垛盘 1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_40">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>460</y>
     <width>100</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>80</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#F1F5FE;
border-style:solid; /*边框风格*/
border-width:1px;/*边框宽度*/
border-color:#B4B6BC; /*边框颜色*/
border-radius:10px; /*边框倒角*/</string>
   </property>
   <property name="text">
    <string>垛盘 4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>110</y>
     <width>150</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>80</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#47E5E8;
border-style:solid; /*边框风格*/
border-width:1px;/*边框宽度*/
border-color:#B4B6BC; /*边框颜色*/
border-radius:10px; /*边框倒角*/</string>
   </property>
   <property name="text">
    <string>端口</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>110</y>
     <width>150</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>80</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#47E5E8;
border-style:solid; /*边框风格*/
border-width:1px;/*边框宽度*/
border-color:#B4B6BC; /*边框颜色*/
border-radius:10px; /*边框倒角*/</string>
   </property>
   <property name="text">
    <string>物品</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_38">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>290</y>
     <width>100</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>80</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#F1F5FE;
border-style:solid; /*边框风格*/
border-width:1px;/*边框宽度*/
border-color:#B4B6BC; /*边框颜色*/
border-radius:10px; /*边框倒角*/</string>
   </property>
   <property name="text">
    <string>垛盘 2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_back">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>620</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    border-style:solid; /*边框风格*/
    border-width:1px;/*边框宽度*/
    border-color:#B4B6BC; /*边框颜色*/
    border-radius:10px; /*边框倒角*/
}
QPushButton:pressed
{
    border-style:outset; /*边框风格*/
    background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop:0 #00befa, stop: 1 #0160ea);/*背景色*/
    border-width:1px;/*边框宽度*/
    border-color:#0055ff; /*边框颜色*/
    border-radius:10px; /*边框倒角*/
}</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/SetPage/setpage_images/back.png</normaloff>:/SetPage/setpage_images/back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>垛盘号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>110</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#47E5E8;
border-style:solid; /*边框风格*/
border-width:1px;/*边框宽度*/
border-color:#B4B6BC; /*边框颜色*/
border-radius:10px; /*边框倒角*/</string>
   </property>
   <property name="text">
    <string>开关</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_port1">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    background-color:#F1F5FE;
    border-style:solid; /*边框风格*/
    border-width:1px;/*边框宽度*/
    border-color:#B4B6BC; /*边框颜色*/
    border-radius:10px; /*边框倒角*/
}
QPushButton:pressed
{
    border-style:outset; /*边框风格*/
    background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop:0 #00befa, stop: 1 #0160ea);/*背景色*/
    border-width:1px;/*边框宽度*/
    border-color:#0055ff; /*边框颜色*/
    border-radius:10px; /*边框倒角*/
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_port2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>290</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    background-color:#F1F5FE;
    border-style:solid; /*边框风格*/
    border-width:1px;/*边框宽度*/
    border-color:#B4B6BC; /*边框颜色*/
    border-radius:10px; /*边框倒角*/
}
QPushButton:pressed
{
    border-style:outset; /*边框风格*/
    background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop:0 #00befa, stop: 1 #0160ea);/*背景色*/
    border-width:1px;/*边框宽度*/
    border-color:#0055ff; /*边框颜色*/
    border-radius:10px; /*边框倒角*/
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_port3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>370</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    background-color:#F1F5FE;
    border-style:solid; /*边框风格*/
    border-width:1px;/*边框宽度*/
    border-color:#B4B6BC; /*边框颜色*/
    border-radius:10px; /*边框倒角*/
}
QPushButton:pressed
{
    border-style:outset; /*边框风格*/
    background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop:0 #00befa, stop: 1 #0160ea);/*背景色*/
    border-width:1px;/*边框宽度*/
    border-color:#0055ff; /*边框颜色*/
    border-radius:10px; /*边框倒角*/
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_port4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>460</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    background-color:#F1F5FE;
    border-style:solid; /*边框风格*/
    border-width:1px;/*边框宽度*/
    border-color:#B4B6BC; /*边框颜色*/
    border-radius:10px; /*边框倒角*/
}
QPushButton:pressed
{
    border-style:outset; /*边框风格*/
    background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop:0 #00befa, stop: 1 #0160ea);/*背景色*/
    border-width:1px;/*边框宽度*/
    border-color:#0055ff; /*边框颜色*/
    border-radius:10px; /*边框倒角*/
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_item1">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>200</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    background-color:#F1F5FE;
    border-style:solid; /*边框风格*/
    border-width:1px;/*边框宽度*/
    border-color:#B4B6BC; /*边框颜色*/
    border-radius:10px; /*边框倒角*/
}
QPushButton:pressed
{
    border-style:outset; /*边框风格*/
    background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop:0 #00befa, stop: 1 #0160ea);/*背景色*/
    border-width:1px;/*边框宽度*/
    border-color:#0055ff; /*边框颜色*/
    border-radius:10px; /*边框倒角*/
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_item2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>290</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    background-color:#F1F5FE;
    border-style:solid; /*边框风格*/
    border-width:1px;/*边框宽度*/
    border-color:#B4B6BC; /*边框颜色*/
    border-radius:10px; /*边框倒角*/
}
QPushButton:pressed
{
    border-style:outset; /*边框风格*/
    background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop:0 #00befa, stop: 1 #0160ea);/*背景色*/
    border-width:1px;/*边框宽度*/
    border-color:#0055ff; /*边框颜色*/
    border-radius:10px; /*边框倒角*/
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_item3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>370</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    background-color:#F1F5FE;
    border-style:solid; /*边框风格*/
    border-width:1px;/*边框宽度*/
    border-color:#B4B6BC; /*边框颜色*/
    border-radius:10px; /*边框倒角*/
}
QPushButton:pressed
{
    border-style:outset; /*边框风格*/
    background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop:0 #00befa, stop: 1 #0160ea);/*背景色*/
    border-width:1px;/*边框宽度*/
    border-color:#0055ff; /*边框颜色*/
    border-radius:10px; /*边框倒角*/
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_item4">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>460</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    background-color:#F1F5FE;
    border-style:solid; /*边框风格*/
    border-width:1px;/*边框宽度*/
    border-color:#B4B6BC; /*边框颜色*/
    border-radius:10px; /*边框倒角*/
}
QPushButton:pressed
{
    border-style:outset; /*边框风格*/
    background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop:0 #00befa, stop: 1 #0160ea);/*背景色*/
    border-width:1px;/*边框宽度*/
    border-color:#0055ff; /*边框颜色*/
    border-radius:10px; /*边框倒角*/
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_OnOff1">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>200</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    background-color:#F1F5FE;
    border-style:solid; /*边框风格*/
    border-width:1px;/*边框宽度*/
    border-color:#B4B6BC; /*边框颜色*/
    border-radius:10px; /*边框倒角*/
}
QPushButton:pressed
{
    border-style:outset; /*边框风格*/
    background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop:0 #00befa, stop: 1 #0160ea);/*背景色*/
    border-width:1px;/*边框宽度*/
    border-color:#0055ff; /*边框颜色*/
    border-radius:10px; /*边框倒角*/
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_OnOff2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>290</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    background-color:#F1F5FE;
    border-style:solid; /*边框风格*/
    border-width:1px;/*边框宽度*/
    border-color:#B4B6BC; /*边框颜色*/
    border-radius:10px; /*边框倒角*/
}
QPushButton:pressed
{
    border-style:outset; /*边框风格*/
    background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop:0 #00befa, stop: 1 #0160ea);/*背景色*/
    border-width:1px;/*边框宽度*/
    border-color:#0055ff; /*边框颜色*/
    border-radius:10px; /*边框倒角*/
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_OnOff3">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>370</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    background-color:#F1F5FE;
    border-style:solid; /*边框风格*/
    border-width:1px;/*边框宽度*/
    border-color:#B4B6BC; /*边框颜色*/
    border-radius:10px; /*边框倒角*/
}
QPushButton:pressed
{
    border-style:outset; /*边框风格*/
    background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop:0 #00befa, stop: 1 #0160ea);/*背景色*/
    border-width:1px;/*边框宽度*/
    border-color:#0055ff; /*边框颜色*/
    border-radius:10px; /*边框倒角*/
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_OnOff4">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>460</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    background-color:#F1F5FE;
    border-style:solid; /*边框风格*/
    border-width:1px;/*边框宽度*/
    border-color:#B4B6BC; /*边框颜色*/
    border-radius:10px; /*边框倒角*/
}
QPushButton:pressed
{
    border-style:outset; /*边框风格*/
    background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop:0 #00befa, stop: 1 #0160ea);/*背景色*/
    border-width:1px;/*边框宽度*/
    border-color:#0055ff; /*边框颜色*/
    border-radius:10px; /*边框倒角*/
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
